src/agent/centralized_configuration/include/centralized_configuration.hpp

```diff
@@ -4,6 +4,8 @@
 
 #include <boost/asio/awaitable.hpp>
 
+#include <nlohmann/json.hpp>
```

Comment:
Now it should be added as a public dependency and linked against it in the cmake file of centralized configuration.

Reply:
Done!